client/my-sites/theme/theme-related-themes-card/index.jsx

```diff
@@ -0,0 +1,86 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import i18n from 'lib/mixins/i18n';
+import SectionHeader from 'components/section-header';
+import { getDetailsUrl } from 'my-sites/themes/helpers';
+
+var ThemeRelatedThemesCard = React.createClass( {
+
+	propTypes: {
+		currentTheme: React.PropTypes.string.isRequired
```

Comment:
using 'slug' rather than 'name' in this property will likely be more useful in future, if we make an API call.